src/BlockTypes/ProductButton.php

```diff
@@ -89,26 +89,49 @@ protected function render( $attributes, $content, $block ) {
 			$html_element                  = ( ! $product->has_options() && $product->is_purchasable() && $product->is_in_stock() && ! $cart_redirect_after_add ) ? 'button' : 'a';
 			$styles_and_classes            = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes, array( 'border_radius', 'font_size', 'font_weight', 'margin', 'padding', 'text_color' ) );
 			$text_align_styles_and_classes = StyleAttributesUtils::get_text_align_class_and_style( $attributes );
+			$html_classes                  = implode(
+				' ',
+				array_filter(
+					array(
+						'wp-block-button__link',
+						'wc-block-components-product-button__button',
+						$product->is_purchasable() && ! $product->has_options() ? 'ajax_add_to_cart add_to_cart_button' : '',
+						'product_type_' . $product->get_type(),
+						$styles_and_classes['classes'],
+					)
+				)
+			);
+
+			$args = apply_filters(
+				'woocommerce_loop_add_to_cart_args',
+				array(
+					'class'      => $html_classes,
+					'attributes' => array(
+						'data-product_id'  => $product->get_id(),
+						'data-product_sku' => $product->get_sku(),
+						'rel'              => 'nofollow',
```

Comment:
Should we mimic the behavior of WC Core here by adding `aria-label` attribute as Core does?